packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -830,6 +830,39 @@ const TESTS = new Map([
       { test: 'notInclude', expected: 'utm_source=email' },
     ]],
   ])],
+  ['postVerifyNonMobileEmail', new Map([
+    ['subject', { test: 'equal', expected: 'Account verified. Next, sync another device to finish setup' }],
+    ['headers', new Map([
+      ['X-Link', { test: 'equal', expected: configUrl('syncUrl', 'account-verified', 'registration-confirmation', 'utm_source=fxa') }],
+      ['X-SES-MESSAGE-TAGS', { test: 'equal', expected: sesMessageTagsHeaderValue('postVerifyNonMobile') }],
+      ['X-Template-Name', { test: 'equal', expected: 'postVerifyNonMobile' }],
+      ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.postVerifyNonMobile }],
+    ])],
+    ['html', [
+      { test: 'include', expected: 'Firefox Account verified. You&#x27;re almost there.' },
+      { test: 'include', expected: 'Next sync between your devices!' },
+      { test: 'include', expected: 'Sync privately keeps your bookmarks, passwords and other Firefox data the same across all your devices.' },
+      { test: 'include', expected: configHref('syncUrl', 'account-verified', 'registration-confirmation', 'utm_source=fxa') },
+      { test: 'include', expected: config.smtp.androidUrl },
```

Comment:
@vbudhram I also have a question here about the `params` const I set, as my tests here are failing since the `androidUrl` and `iosUrl` both have `utm_source=fxa` appended. I'm sure I can fix that, but, is it fine that `utm_source=fxa` is appended to all URLs or should I figure out how to target _only_ the `connect another desktop device` link?